.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -361,71 +388,93 @@ jobs:
-          # Randomly select an item from assignees list
-          ASSIGNEE=${ASSIGNEES_LIST[ $(( RANDOM % ${#ASSIGNEES_LIST[@]} )) ]}
-          echo "::set-output name=assignee::$ASSIGNEE"
-
-      - name: Open GitHub Issue 📬
-        if: steps.performance.outputs.is_dropped == 'true'
-        uses: rishabhgupta/git-action-issue@v2
-        id: open_issue
+        uses: actions/github-script@v3
+        id: issue-exist
         with:
-            token: ${{ secrets.GITHUB_TOKEN }}
-            title: "Scheduled Model Regression Test Performance Drops"
-            body: "*This PR is automatically created by the Scheduled Model Regression Test workflow. Checkout the Github Action Run [here](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}).* <br> --- <br> **Description of Problem:** <br>  Some test performance scores **decreased**. Please look at the following table for more details. <br> ${{ steps.performance.outputs.report_description }}"
-            assignees: ${{ steps.pick_assignee.outputs.assignee }}
+          result-encoding: string
+          github-token: ${{ github.token }}
+          script: |
+            // Get all open issues
+            const opts = await github.issues.listForRepo.endpoint.merge({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open',
+              labels: ${{ env.GITHUB_ISSUE_LABEL }}
+            });
+            const issues = await github.paginate(opts)
+            // Check if issue exist
+            for (const issue of issues) {
+              if (issue.title.includes('${{ env.GITHUB_ISSUE_TITLE }}') ) {
+                  console.log(`Found an exist issue \#${issue.number}. Skip the following steps.`);
+                  return 'true'
+                }
+            }
+            return 'false'
 
-      - name: Add Label to the Issue
-        if: steps.performance.outputs.is_dropped == 'true'
-        id: add_label
+      - name: Select a member from managers team randomly
+        if:  steps.performance.outputs.is_dropped == 'true' && steps.issue-exist.outputs.result == 'false'
         run: |
-          # Get issue number
-          ISSUE_NUMBER=$(echo '${{ steps.open_issue.outputs.issue }}' | jq -r '.number')
-          echo "::set-output name=issue_number::$ISSUE_NUMBER"
-
-          # Add label to the issue
-          curl \
-            -X POST \
-            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-            -H "Accept: application/vnd.github.v3+json" \
-            https://api.github.com/repos/$GITHUB_REPOSITORY/issues/${ISSUE_NUMBER}/labels \
-            -d '{"labels": ${{ env.GITHUB_ISSUE_LABELS }}}'
-
-      - name: Notify Slack when Performance Drops 💬
-        if: steps.performance.outputs.is_dropped == 'true'
-        uses: 8398a7/action-slack@v3
+          # MEMBERS=$(curl \
+          #             -H "Authorization: token 58290fcf8a21bec2ce47515a8b5b86c8ce95064e" \
```

Comment:
@virtualroot When you have time tomorrow, could you help me to replace this to `${{secrets.GITHUB_TOKEN}}`. 🙏🏻